.github/workflows/build.yaml

```diff
@@ -16,8 +16,12 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
-      - name: Build Binary via Docker
-        run: docker build -t kubemarine --build-arg BUILD_TYPE=binary --no-cache .
+      - name: PyInstaller amd64
+        id: amd64
+        uses: action-python/pyinstaller-py3.8@amd64
+        with:
+          path: ./
+          requirements: "./requirements.txt"
```

Comment:
We have selftest of docker image and it will be good to add selftest for binaries as well.

Reply:
add selftest for binary files